phalcon/mvc/model.zep

```diff
@@ -98,7 +98,7 @@ abstract class Model implements EntityInterface, ModelInterface, ResultInterface
 
 	protected _dirtyState = 1;
 
-	protected _transaction;
+	protected _transaction { get };
```

Comment:
The`_transaction` property now has a public method here. So could you please add  
```zep
/**
 * @var TransactionInterface | null
 */
```